src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Items/xlf/EditorConfigFiles.BrowseObject.xaml.ru.xlf

```diff
@@ -4,97 +4,97 @@
     <body>
       <trans-unit id="Rule|EditorConfigFiles|Description">
         <source>File Properties</source>
-        <target state="new">File Properties</target>
+        <target state="translated">Свойства файла</target>
         <note />
       </trans-unit>
       <trans-unit id="Rule|EditorConfigFiles|DisplayName">
         <source>.editorconfig File</source>
-        <target state="new">.editorconfig File</target>
+        <target state="translated">Файл EDITORCONFIG</target>
```

Comment:
Another capitalisation of `.editorconfig`.

Reply:
Answered in above comment.